### PR TITLE
[bitnami/consul] Disallowing privilege escalation

### PR DESCRIPTION
**Pull request by vinokuma**
Signed-off-by: vinokuma <vinod.kumar@radisys.com>

### Description of the change

In this PR, we have added one parameter `allowPrivilegeEscalation: false` to `values.yaml` in `consul` to disallow privilege escalation

### Benefits

a child process should not be able to get more privileges than its parent process, e.g. via set uid bit. This should be fixed by adding securityContext to the deployment yaml of the pod

### Applicable issues

  - fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for your contribution. Could you please take a look at the failing action? You need to add the proper metadata for the new parameter

**Comment by vinokuma** [comment]
> Thanks for your contribution. Could you please take a look at the failing action? You need to add the proper metadata for the new parameter

Yes, made the changes and the build is pass now

**Comment by vinokuma** [comment]
Hi @carrodher, can you please review this PR and approve if its fine ?

File: bitnami/consul/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: consul
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/consul
   - https://www.consul.io/
-version: 10.8.2
+version: 10.9.2
```
Comment (carrodher) [inline bitnami/consul/Chart.yaml:26]: Thanks for the PR, please note there is an issue with the version bump; it should be `10.9.0`

Comment (vinokuma) [reply bitnami/consul/Chart.yaml]: Thanks, changed it to 10.9.0

**Comment by vinokuma** [comment]
Hi @carrodher, could you please review this PR and approve if its fine ?

**Comment by vinokuma** [comment]
Hi @carrodher, could you please review this PR and approve if its fine ?

**Review by carrodher — APPROVED** [review]

**Pull request merged**